config/makefile

```diff
@@ -110,4 +110,5 @@ config-mk:
 
 clean:
 	rm -f $(CONFIG_HPP) $(CONFIG_MK) sample-runs-build.log
-	rm -f $(GHV) $(GHV).out $(GMV) $(GMV).out *.dSYM
+	rm -f $(GHV) $(GHV).out $(GMV) $(GMV).out
+	rm -rf  *.dSYM
```

Comment:
```suggestion
	rm -rf *.dSYM
```

Reply:
done in https://github.com/mfem/mfem/pull/4716/commits/31f3ec78ddbfb9cb4a0f84476aa5d8581112b236